mscore/scoretreeview.h

```diff
@@ -0,0 +1,42 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __STREEVIEW_H__
+#define __STREEVIEW_H__
+
+#include <QtWidgets>
+#include "musescore.h"
+#include "libmscore/score.h"
+#include "scoreitemmodel.h"
+
+namespace Ms {
+
+class ScoreTreeView : public QObject
+{
+    Q_OBJECT
+
+    QDockWidget* _dock;
+    QTreeView* _treeView;
+    Score* _score;
+
+public:
+    ScoreTreeView();
```

Comment:
All Qt-derived objects should accept a parent object in their constructor. i.e.:

```
ScoreTreeView(QObject* parent=nullptr) : QObject(parent) {}
```

For widgets (like QDockWidget), you can assume the parent is also a widget (but not necessarily the same kind):

```
ScoreTreeView(QWidget* parent=nullptr) : QDockWidget(parent) {}
```

Notice the parent can be any QWidget, not necessarily a QDockWidget.